### PR TITLE
fix:Replacing-no-posts-yet-text

### PR DESCRIPTION
**Pull request by jatinder14**
Fixes: #4735

### CODE REVIEW

**Comment by jatinder14** [comment]
Fixes https://github.com/ushahidi/platform/issues/4735#event-12029371102

**Comment by jatinder14** [comment]
Outreachy Task Submission

**Review by Angamanga — CHANGES_REQUESTED** [review]
Thank you for your PR @jatinder14! Could you double-check so that string you changed are not used in other instances, for example, when there are actually no posts available? I suspect it is used in more cases, for example when there are no posts in a deployment, and then the text is actually correct. So there needs to be some logic added for when to show what text.

**Comment by jatinder14** [comment]
@Angamanga can you please check it again ?

File: apps/web-mzima-client/src/app/feed/feed.component.html [hunk 1]
```diff
@@ -272,5 +272,10 @@ <h3 class="menu-title">{{ 'app.mark_as' | translate }}</h3>
 </ng-template>
 
 <ng-template #postsEmpty>
-  <span class="posts-empty">{{ 'post.no_posts_yet' | translate }}</span>
+  <span class="posts-empty" *ngIf="filters.form[0] === 'none'; else noPosts">{{
+    'post.no_posts_yet' | translate
+  }}</span>
+  <ng-template #noPosts>
+    <span class="posts-empty">There are no posts in the survey!</span>
```
Comment (Angamanga) [inline apps/web-mzima-client/src/app/feed/feed.component.html]: Add the string to the en.json-file instead and map it in a similar way as is already done above, on row 276

**Review by Angamanga — CHANGES_REQUESTED** [review]
Great start! See my comment in the code for directions 👍

File: apps/web-mzima-client/src/app/feed/feed.component.html (hunk 1 above)
Comment (jatinder14) [reply apps/web-mzima-client/src/app/feed/feed.component.html]: alright @Angamanga But it should be done for english only or for all the available languages?

**Comment by jatinder14** [comment]
@Angamanga  can you please check it again ?

**Comment by jatinder14** [comment]
@Angamanga

**Review by Angamanga — APPROVED** [review]
Great work! Approved 👍

**Comment by Angamanga** [comment]
@Angamanga To consider for next release

**Comment by jatinder14** [comment]
please merge it @Angamanga

**Comment by jatinder14** [comment]
Hi @Angamanga @tuxpiper

**Comment by jatinder14** [comment]
@Angamanga

**Comment by Angamanga** [comment]
Fixed

**Pull request closed without merging**